besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -1236,28 +1200,6 @@ public void parsesValidSnapSyncMinPeersOption() {
     assertThat(commandErrorOutput.toString(UTF_8)).isEmpty();
   }
 
-  @Test
-  public void parsesValidSyncMinPeersOption() {
-    parseCommand("--sync-mode", "FAST", "--sync-min-peers", "11");
```

Comment:
if this test is not duplicated for SNAP, I would change it over rather than removing